src/bidders/prebid/index.js

```diff
@@ -0,0 +1,151 @@
+import { context, utils } from '@wikia/ad-engine';
+import { BaseBidder } from './../base-bidder';
+import { universalAdPackage } from './../../templates/uap/universal-ad-package';
+import { getPriorities } from './adapters-registry';
+import { getPrebidBestPrice } from './price-helper';
+import { getSettings } from './prebid-settings';
+import { setupAdUnits } from './prebid-helper';
+
+export class Prebid extends BaseBidder {
+	constructor(bidderConfig, resetListener, timeout = 2000) {
+		super(bidderConfig, resetListener, timeout);
+
+		this.logGroup = 'prebid-bidder';
+		this.name = 'prebid';
+		this.loaded = false;
+		this.lazyLoaded = false;
+		this.isLazyLoadingEnabled = this.bidderConfig.lazyLoadingEnabled;
+		this.isCMPEnabled = context.get('custom.isCMPEnabled');
```

Comment:
s/custom.isCPMEnabled/options.prebid.isCPMEnabled
or bidders.prebid.isCPMEnabled

Reply:
I think it's more general option by design, what do we want to store it under prebid?

Reply:
I agree with Maciej, we should leave it as it is.